docs/supportedchains/supportedTokens.md

```diff
@@ -5,9 +5,9 @@ sidebar_position: 2
 
 # ERC20 Gas Payments
 
-Our Paymaster not only allows you to sponsor gasless tokens for your users but also allows for users to pay for tokens in supported ERC20 tokens. Below are supported tokens grouped by chain.
+Our Paymaster not only allows you to sponsor transactions for your users but also allows users to pay for transactions using ERC20 tokens. Below are supported tokens list grouped by chain.
 
-Please note that we will be reviewing DEX activitiy on each network to determine additional ERC20 token supoort on an ongoing basis.
+Please note that we will be reviewing DEX activity on each network to determine additional ERC20 token support on an ongoing basis.
```

Comment:
we should add

Want to add a token? Please [contact us](mailto:support@biconomy.io) and we'll see what we can do.

do confirm contact email